Libraries/Opc.Ua.Client/Session/TraceableSession.cs

```diff
@@ -1265,6 +1265,30 @@ public async Task<BrowseNextResponse> BrowseNextAsync(RequestHeader requestHeade
             }
         }
 
+
+        /// <inheritdoc/>
+        public void ManagedBrowse(RequestHeader requestHeader, ViewDescription view, IList<NodeId> nodesToBrowse, uint maxResultsToReturn, BrowseDirection browseDirection, NodeId referenceTypeId, bool includeSubtypes, uint nodeClassMask, out List<ReferenceDescriptionCollection> result, out List<ServiceResult> errors, bool excecuteDefensively = false)
+        {
+            using (Activity activity = ActivitySource.StartActivity())
+            {
+                m_session.ManagedBrowse(requestHeader, view, nodesToBrowse, maxResultsToReturn, browseDirection, referenceTypeId, includeSubtypes, nodeClassMask, out result, out errors, excecuteDefensively);
+            }
+
+        }
+        
+        /// <inheritdoc/>        
+        public async Task<(
+            List<ReferenceDescriptionCollection>,
+            IList<ServiceResult>
+            )>
+                ManagedBrowseAsync(RequestHeader requestHeader, ViewDescription view, IList<NodeId> nodesToBrowse, uint maxResultsToReturn, BrowseDirection browseDirection, NodeId referenceTypeId, bool includeSubtypes, uint nodeClassMask, bool executeDefensively = false, CancellationToken ct = default)
```

Comment:
something is not quite right with the formatting...

Reply:
fixed